shap/plots/_heatmap.py

```diff
@@ -136,7 +145,7 @@ def heatmap(shap_values, instance_order=Explanation.hclust(), feature_values=Exp
 
     # plot the f(x) line chart above the heat map
     ax.axhline(-1.5, color="#aaaaaa", linestyle="--", linewidth=0.5)
-    fx = values.T.mean(0)
+    fx = values.T.sum(0)
```

Comment:
This would change existing plots. Would you be so kind to explain why you'd like to change this?

Reply:
I thought so too, but it seems not to change the plot due to the abs_max normalization before plotting. I shortly discussed this in #3544.

This works because of two operations. First, the `mean()` and `sum()` are computed over the same number of samples, and second, the values are scaled by a [max_abs scaling](https://github.com/shap/shap/blob/dbb275978abe27def1afe73fdd097d071c7fbe38/shap/plots/_heatmap.py#L141C1-L146C6) afterwards, bringing the resulting values to a common scale.

Because SHAP should be additive, I thought the `sum()` operation would be more intuitive.